src/execution/execute.js

```diff
@@ -1247,7 +1247,8 @@ function (source, args, context, info) {
   if (typeof source === 'object' || typeof source === 'function') {
     const property = source[info.fieldName];
     if (typeof property === 'function') {
-      return source[info.fieldName](args, context, info);
+      /* eslint-disable no-useless-call */
+      return source[info.fieldName].apply(source, [ args, context, info ]);
```

Comment:
Function#call is [usually faster](https://jsperf.com/function-calls-direct-vs-apply-vs-call-vs-bind/6) than Function#apply, and since you're not passing varargs here, it'd probably be better to go:

```js
return source[info.fieldName].call(source, args, context, info);
```